source/includes/Person/persons.md

```diff
@@ -144,24 +145,132 @@ DELETE /person/:target_person_uuid?purge=true
 
 ## List person name subresource
 
-```console
+> List person name subresource
+
+```shell
 GET /person/:target_person_uuid/name
 ```
-* List all the person name subresource corresponding to a `target_person_uuid`. Returns `404 Not Found` status if the person does not exist. 
-If not authenticated or authenticated user does not have sufficient privileges, a `401 Unauthorized` 
-status is returned.
 
-```console
+> Success Response
+
+```response
+
+{
+    "results": [
+        {
+            "display": "John Taylor",
+            "uuid": "4b68f067-6f4d-451a-bd80-342fc21ea486",
+            "givenName": "John",
+            "middleName": null,
+            "familyName": "Taylor",
+            "familyName2": null,
+            "voided": false,
+            "links": [
+                {
+                    "rel": "self",
+                    "uri": "http://qa-refapp.openmrs.org/openmrs/ws/rest/v1/person/90f7f0b4-06a8-4a97-9678-e7a977f4b518/name/4b68f067-6f4d-451a-bd80-342fc21ea486",
+                    "resourceAlias": "name"
+                },
+                {
+                    "rel": "full",
+                    "uri": "http://qa-refapp.openmrs.org/openmrs/ws/rest/v1/person/90f7f0b4-06a8-4a97-9678-e7a977f4b518/name/4b68f067-6f4d-451a-bd80-342fc21ea486?v=full",
+                    "resourceAlias": "name"
+                }
+            ],
+            "resourceVersion": "1.8"
+        }
+    ]
+}
+
+```
+
+```java
+
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+Request request = new Request.Builder()
+  .url("https://qa-refapp.openmrs.org/openmrs/ws/rest/v1/person/90f7f0b4-06a8-4a97-9678-e7a977f4b518/name")
+  .method("GET", null)
```

Comment:
@Akayeshmantha I saw you made some changes to all the previous files and shortened the URL removing the refapp or demo server, will the examples still work or we will have to add a note to the user so that they can add specific server base url by themselves?

Reply:
since I tried making examples which could run against the demo-server should we append something so that user knows they have to fill in the base URL?